content/influxdb/clustered/install/licensing.md

```diff
@@ -1,114 +1,104 @@
 ---
-title: Install Your InfluxDB 3.0 License
+title: Install your InfluxDB Clustered license
 description: >
-  Install the InfluxDB 3.0 License in your cluster.
+  Install your InfluxDB Clustered license to authorize the use of the InfluxDB
+  Clustered software.
 menu:
   influxdb_clustered:
     name: Install your License
     parent: Install InfluxDB Clustered
 weight: 135
+influxdb/clustered/tags: [licensing]
 related:
-  - /influxdb/clustered/install/deploy/
-  - /influxdb/clustered/install/configure-cluster/
-  - /influxdb/clustered/reference/licensing/
+  - /influxdb/clustered/admin/licensing/
+  - /influxdb/clustered/admin/upgrade/
 ---
 
-This document outlines the installation process for licenses in InfluxDB 3.0
-Clustered. For information about how license enforcement works, please see the
-[Licensing Enforcement reference doc][enforcement], which explains the license
-enforcement behaviors you should expect from the product.
+Install your InfluxDB Clustered license in your cluster to authorize the use
+of the InfluxDB Clustered software.
 
 {{% note %}}
+#### License enforcement is currently an opt-in feature
 
-We are entering a transition period, during which InfluxDB 3.0 Clustered
-license enforcement is an opt-in feature. This transition period allows us to
-gently introduce license enforcement to a select few customers in a way that
-can easily be disabled if problems arise. After the end of this transition
-period, all releases of InfluxDB 3.0 Clustered will require an active license
-for the product.
+In currently available versions of InfluxDB Clustered, license enforcement is an
+opt-in feature. This allows InfluxData to introduce license enforcement to a
+limited number of customers in a way that can easily be disabled if issues arise.
```

Comment:
- Suggestion: use "turned off" or "deactivated" in place of "disabled" (see https://developers.google.com/style/word-list#disable)
- Who can (or is allowed to) deactivate it, the user or InfluxData?
- Can the customer opt out?